lib/v8debugapi.js

```diff
@@ -108,21 +123,22 @@ module.exports.create = function(logger_, config_, fileStats_) {
           StatusMessage.UNSPECIFIED, messages.INVALID_BREAKPOINT);
       }
 
-      var scriptPath = path.normalize(breakpoint.location.path);
-      var compile = null;
+      var baseScriptPath = path.normalize(breakpoint.location.path);
+      if (!sourceMapper.hasMappingInfo(baseScriptPath)) {
+        if (!_.endsWith(baseScriptPath, '.js')) {
+          return setErrorStatusAndCallback(cb, breakpoint,
+                  StatusMessage.BREAKPOINT_SOURCE_LOCATION,
+                  messages.COULD_NOT_FIND_OUTPUT_FILE);
+        }
 
-      if (endsWith(scriptPath, '.js')) {
-        setInternal(breakpoint, scriptPath, compile, cb);
-      } else {
-        var sm = require('source-map');
-        var fs = require('fs');
-        var sourcePos = {
-          source: scriptPath.indexOf(path.sep) === -1 ? scriptPath
-            : scriptPath.split(path.sep).pop(),
-          line: breakpoint.location.line,
-          column: 0 // unlike lines, columns are 0-indexed for source-map ಠ_ಠ.
-        };
-        compile = getBreakpointCompiler(breakpoint);
+        setInternal(breakpoint, null /* mapInfo */, null /* compile */, cb);
+      }
+      else {
+        var line = breakpoint.location.line,
+            column = 0,
+            mapInfo = sourceMapper.mappingInfo(baseScriptPath, line, column);
```

Comment:
style nit: use multiple var statements to be consistent with the rest of the project.